modules/@apostrophecms/page/ui/apos/components/AposPagesOrganizer.vue

```diff
@@ -0,0 +1,218 @@
+<template>
+  <AposModal
+    :modal="modal" modal-title="Manage Page Tree"
+    @esc="cancel" @no-modal="$emit('safe-close')"
+    @inactive="modal.active = false" @show-modal="modal.showModal = true"
+  >
+    <template #primaryControls>
+      <AposButton
+        type="default" label="Finished"
+        @click="cancel"
+      />
+    </template>
+    <template #main>
+      <AposModalBody>
+        <template #bodyHeader>
+          <AposPagesOrganizerToolbar
+            :selected-state="selectAllState"
+            @select-click="selectAll"
+            @trash-click="trashClick"
+          />
+        </template>
+        <template #bodyMain>
+          <AposTree
+            :rows="rows"
+            :headers="headers" :icons="icons"
+            :draggable="true" :selectable="true"
+            v-model="checked"
+            @update="update" @busy="setBusy"
+            @edit="openEditor"
+          />
+        </template>
+      </AposModalBody>
+    </template>
+  </AposModal>
+</template>
+
+<script>
+import AposModalParentMixin from 'Modules/@apostrophecms/modal/mixins/AposModalParentMixin';
+import AposTableMixin from 'Modules/@apostrophecms/modal/mixins/AposTableMixin';
+
+export default {
+  name: 'AposPagesOrganizer',
```

Comment:
We moved away from "Reorganize" in the UI and toward treating "manage pages" like other things, so I'm not sure if this is the right name for the component if it is analogous to the pieces manager.